docs/tutorials/getting_started/initialize_a_data_context.rst

```diff
@@ -24,54 +24,45 @@ or git, you may want to check out the :ref:`supporting_resources` section before
 
     $ pip install great_expectations
 
-To install from a git branch:
+To install from a git branch, use the following command (replace ``develop`` below with the name of the branch you want to use):
 
 .. code-block:: bash
 
-    #FIXME:
     $ git clone https://github.com/great-expectations/great_expectations.git
-    $ pip install great_expectations/
+    $ cd great_expectations/
+    $ git checkout develop
+    $ pip install -e .
 
-To install from a git fork:
+To install from a git fork, use the following command (replace ``great-expectations`` below with the name of the fork, which is usually your github username):
 
 .. code-block:: bash
 
+    $ pip install -e .
     $ git clone https://github.com/great-expectations/great_expectations.git
     $ pip install great_expectations/
 
 If you intend to develop within the Great Expectations (e.g. to contribute back to the project), check out :ref:`contributing_setting_up_your_dev_environment` in the contributor documentation.
 
-Get the ``example-dickens-data-project``
+Downloading Example Data ** to fix?
```

Comment:
```suggestion
Downloading Example Data
```

Reply:
yup should be addressed in the next commit